src/views/ScheduledRestock.vue

```diff
@@ -0,0 +1,437 @@
+            <template v-if="field === 'productIdentification'">
+              <ion-select aria-label="identification-type-id" interface="popover" :placeholder = "translate('Select')" v-model="identificationTypeId">
+                <ion-select-option :key="goodIdentificationType.goodIdentificationTypeId" :value="goodIdentificationType.goodIdentificationTypeId" v-for="goodIdentificationType in goodIdentificationTypes">{{ goodIdentificationType.description }}</ion-select-option>
+              </ion-select>
+              <ion-select aria-label="identification-type-value" interface="popover" :disabled="!content.length" :placeholder = "translate('Select')" slot="end" v-model="fieldMapping['productIdentification']">
+                <ion-select-option :key="index" v-for="(prop, index) in fileColumns">{{ prop }}</ion-select-option>
+              </ion-select>
+            </template>
+            <template v-else>
+              <ion-select :label="translate(fieldValues.label)" interface="popover" :disabled="!content.length" :placeholder = "translate('Select')" v-model="fieldMapping[field]">
+                <ion-select-option :key="index" v-for="(prop, index) in fileColumns">{{ prop }}</ion-select-option>
+              </ion-select>
+            </template>
+          </ion-item>
+
+          <ion-item-divider>
+            <ion-label>{{ translate("Optional, or select during review") }}</ion-label>
+          </ion-item-divider>
+          <ion-item>
+            <ion-label>{{ translate("Schedule") }}</ion-label>   
+            <ion-button slot="end" class="date-time-button" @click="updateTime()">{{ schedule ? getTime(schedule) : translate("Select time") }}</ion-button>
+            <ion-modal class="date-time-modal" :is-open="isDateTimeModalOpen" @didDismiss="() => isDateTimeModalOpen = false">
+              <ion-content force-overscroll="false">
+                <ion-datetime    
+                  id="schedule-datetime"        
+                  show-default-buttons 
+                  hour-cycle="h23"
+                  presentation="date-time"
+                  :value="schedule ? getDateTime(schedule) : getDateTime(DateTime.now().toMillis())"
+                  @ionChange="updateCustomTime($event)"
+                />
+              </ion-content>
+            </ion-modal>
+          </ion-item>
+          <ion-item>
+            <ion-select :label="translate('Facility')" interface="popover" :placeholder = "translate('Select')" v-model="selectedFacility">
+              <ion-select-option v-for="facility in facilities" :key="facility.facilityId" :value="facility.externalId">
+                {{ facility.facilityName || facility.facilityId }}
+              </ion-select-option>
+            </ion-select>
+          </ion-item>
+          <ion-item>
+            <ion-select :label="translate('Product store')" interface="popover" :value="selectedProductStoreId" @ionChange="updateProductStore($event)">
```

Comment:
Add a placeholder here as well, as you have added placeholders on facility and shopifyShop option